toucan_connectors/mysql/mysql_connector.py

```diff
@@ -159,14 +160,21 @@ def ssl_key_validator(cls, ssl_key: str, values: dict) -> str:
 
         return ssl_key
 
+    # - fix le parsing des PEM
+    # - _sanitize_ssl_params -> check si la string est pas None au lieu de vide
+
     def _sanitize_ssl_params(self) -> dict[str, Any]:
         params = {}
         if self.ssl_mode in (SSLMode.VERIFY_CA, SSLMode.VERIFY_IDENTITY):
 
             for ssl_opt in ('ssl_ca', 'ssl_key', 'ssl_cert'):
                 try:
-                    if (opt := getattr(self, ssl_opt)) is not None:
-                        params[ssl_opt] = sanitize_spaces_pem(opt.get_secret_value())
+                    opt = getattr(self, ssl_opt)
+                    if opt is None:
+                        continue
+                    secret = opt.get_secret_value()
+                    if secret != '':
```

Comment:
```suggestion
                    if secret.strip() != '':
```

Juste pour être OK sur le whitespace